js/ui/codingManager.js

```diff
@@ -359,6 +361,84 @@ const WindowTrackingButton = new Lang.Class({
     }
 });
 
+const CodingInstallationMonitor = new Lang.Class({
+    Name: 'CodingInstallationMonitor',
+    Extends: GObject.Object,
+    Properties: {
+        'flatpak-name': GObject.ParamSpec.string('flatpak-name',
```

Comment:
This should be called `flatpak-id` or something like that -- `name` usually implies something user-visible.